content/cloud-docs/run/cli.mdx

```diff
@@ -47,9 +47,11 @@ To supplement these remote operations, you can also use the optional [Terraform
 
 To enable the CLI-driven workflow, you must:
 
-1. Run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file](/cli/config/config-file#credentials).
+1. Run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file](/cli/config/config-file#environment-variable-credentials.
```

Comment:
```suggestion
1. Run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file or through environment variables](/cli/config/config-file#environment-variable-credentials.
```

Reply:
Okay now that link is a bit too long. I'm going to remove the long in-text link and add a "refer to" statement instead.